pilot/pkg/model/push_context.go

```diff
@@ -95,9 +96,19 @@ type PushContext struct {
 	// ServiceAccounts contains a map of hostname and port to service accounts.
 	ServiceAccounts map[host.Name]map[int][]string `json:"-"`
 
+	// AuthNPolicies contains a map of hostname and port to authentication policy
+	AuthNPolicies map[host.Name]map[int][]*Config `json:"-"`
```

Comment:
May be use `authn.Policy` instead of Config?

Reply:
AuthnPolicies